docs/managing-images.md

```diff
@@ -0,0 +1,81 @@
+# Managing images
+
+## Using private Docker registries
+
+FaaS supports running functions from Docker images in private Docker registries.
+The registry credentials can be passed on function deployment, and are then handled by Swarm for image polling.
+
+### Deploy functions with private registries credentials
+
+A `POST` request on `/system/function` allows you to specify private registry credentials, as a base64-encoded basic auth (user:password).
+```
+curl -XPOST /system/functions -d {
+    "service": "functionName",
+    "image": "privateregistry.domain.com/user/function",
+    "envProcess": "/usr/bin/myprocess",
+    "network": "func_functions",
+    "registryAuth": "dXNlcjpwYXNzd29yZAo"
+}
+```
+
+Base64-encoded basic auth can be computed manually, or found in your `~/.docker/config.json` Docker credentials store, as a result of the `docker login`command:
```

Comment:
"computed manually".. an example would be what - `echo "username:password" | base64`?